dev/openchpl_data-model.sql

```diff
@@ -3499,6 +3384,7 @@ CREATE TABLE openchpl.certification_criterion_attribute (
   CONSTRAINT certification_criterion_id_fk FOREIGN KEY (criterion_id)
         REFERENCES openchpl.certification_criterion (certification_criterion_id)
         MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT
+);
```

Comment:
Where is this coming from?

Reply:
For some reason I saw it was missing.  Not sure where/when it happened, but figured I'd just fix it.